_playwright-tests/UI/CustomRepoPagination.spec.ts

```diff
@@ -0,0 +1,61 @@
+import { test, expect } from '@playwright/test';
+import { navigateToRepositories } from './helpers/navHelpers';
+import { deleteAllRepos } from './helpers/deleteRepositories';
+import { bulkCreateRepos } from './helpers/createRepositories';
+import { closePopupsIfExist, getRowByNameOrUrl } from './helpers/helpers';
```

Comment:
I don't think `getRowByNameOrUrl` is used. Lint is failing:
> 'getRowByNameOrUrl' is defined but never used  @typescript-eslint/no-unused-vars

Reply:
Hi, fixed, thanks